config/default/filter.format.minimal_html.yml

```diff
@@ -17,7 +17,7 @@ filters:
     status: true
     weight: -10
     settings:
-      allowed_html: '<em> <strong> <cite> <blockquote cite> <code> <ul type> <ol start type> <li> <dl> <dt> <dd> <h2 id> <h3 id> <h4 id> <h5 id> <h6 id> <hr> <p class> <span class> <table class> <a href hreflang data-entity-substitution data-entity-type data-entity-uuid title class>'
+      allowed_html: '<em> <strong> <cite> <blockquote cite> <code> <ul type> <ol start type> <li> <dl> <dt> <dd> <h2 id> <h3 id> <h4 id> <h5 id> <h6 id> <hr> <p class=""> <span class=""> <table class=""> <a href hreflang data-entity-substitution data-entity-type data-entity-uuid title class=" hs-button hs-button--big hs-secondary-button hs-more-link hs-external-link hs-file-link hs-pdf-link hs-private-link">'
```

Comment:
This change is not necessary. it limits the classes that can be used on the elements. by specifying `class` alone, it allows any value for that attribute. if you provide `class="foo-bar"` the only class that is available is the `foo-bar` class. I'd rather allow any class. It also could have implications on the display of existing sites, including on the older themes.

Reply:
@pookmish I've added a commit to revert this change. :smile: Thanks!

Reply:
Thanks